working-with-your-data/template-scripts/wget-TEMPLATE.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This is a template script for downloading data using the wget command
+# See docs here: https://www.gnu.org/software/wget/manual/wget.html
+
+mkdir <FOLDER_TO_SAVE_TO>
+
+# To see wget options, use -h (the help flag)
+wget -h
+
+wget -O <FOLDER/FILE_TO_SAVE_TO> <URL>
+
+# Examples of if you have a username and password:
+# wget -O <FILE_PATH_TO_SAVE_TO> ftp://user:password@host/path
+# wget -O <FILE_PATH_TO_SAVE_TO> http://user:password@host/path
```

Comment:
I don't know if we want to make this recommendation because I believe if you were to run `htop` you can see someone's password, for example.  Thoughts @jashapiro ?

Reply:
Yes, generally better to have people enter passwords interactively, even though this is a pain for scripts. I don't think a script will expose the password to `htop`, but then the password is written down in an unencrypted file (with likely insecure permissions). 

Security is hard.

Reply:
Yuck. Hm. So what do we want to recommend instead?

Reply:
Probably running interactively - something like if you get this error then do X. Although we may not want that in the script, the Markdown might be a better spot for it.